api/src/test/java/com/basistech/rosette/api/RosetteAPITest.java

```diff
@@ -301,7 +317,7 @@ public void testNonIgnoredUnknownField() throws IOException {
 
     private void verifyEntity(EntitiesResponse response) throws IOException {
         EntitiesResponse goldResponse = mapper.readValue(responseStr, EntitiesResponse.class);
-        assertEquals(response.getEntities().size(), goldResponse.getEntities().size());
+        assertThat(response.getEntities(), is(goldResponse.getEntities()));
```

Comment:
so a shallow equal is ok here?

Reply:
Think so. I'll fix. We need a "strict mode" test to check valid mappings somewhere, at least to check integrity of tests. Production behavior seems too loose to break the test.